nncf/torch/quantization/algo.py

```diff
@@ -1361,6 +1362,11 @@ def __init__(
             "export_to_onnx_standard_ops", QUANTIZATION_EXPORT_TO_ONNX_STANDARD_OPS
         )
         if should_export_to_onnx_qdq:
+            warning_deprecated(
+                "The config option `export_to_onnx_standard_ops` is deprecated and will be removed in a future version. "
+                "Please use the `nncf.strip(quantized_model)` method instead to export to ONNX "
```

Comment:
`.strip()` does not do export on its own. Specify just when and in what context the export should occur, i.e. "Call `quantized_model.nncf.strip()` before calling `compression_ctrl.export_model(...)` or `torch.onnx.export(...)` to export to ONNX standard ops"

Reply:
Yes, thanks. Rewrite as `Please use the `nncf.strip(quantized_model)` method before export to ONNX to get model with QuantizeLinear-DequantizeLinear node pairs.`
But still write `nncf.strip` because it's common api for PTQ and QAT model.